### PR TITLE
Make more test models smaller

### PR DESCRIPTION
**Pull request by sgugger**
# What does this PR do?

This PR continues on the work of #24824 by fixing the models used in more common tests. This is big enough to be reviewed, I'll do the last ones in a separate PR :-)

### CODE REVIEW

File: tests/test_modeling_common.py [hunk 1]
```diff
@@ -2708,7 +2708,7 @@ def test_load_with_mismatched_shapes(self):
     def test_model_is_small(self):
         # Just a consistency check to make sure we are not running tests on 80M parameter models.
         config, _ = self.model_tester.prepare_config_and_inputs_for_common()
-        # print(config)
+        print(config)
```
Comment (ydshieh) [inline tests/test_modeling_common.py:2711]: is this to be kept when merged?

**Review by ydshieh — APPROVED** [review]
Thank you for all the efforts. Other than a comment about `print`, I am just wondering the numbers used is just to keep `num_params < 1000000`, but in theory we can go even smaller? (no need to change, just curious)

File: tests/test_modeling_common.py (hunk 1 above)
Comment (sgugger) [reply tests/test_modeling_common.py:2711]: Nope,oops!

**Comment by sgugger** [comment]
Yes we could go even smaller if we wanted to, but it's hard for big encoer/decoder models with backbones.

**Pull request merged**